### ISSUE TITLE
Grouping of result columns

### ISSUE THREAD

**Issue by negbie** [sipcapture/homer-app#216]
In Homer 5 it was possible to group search result columns. This was quite handy when you use Timeseries Monitoring Systems where you i.e. see how many bad requests comes from a certain Proxy but you don't know exactly if just one user or multiple user causing them. To group by from or to user showed you this very quickly.

**Comment by lmangani** [comment]
Good Point, more about the library we're using for grid than any other real choice at this time.

In parallel, we're trying to complement this with the "results widgets" which allow displaying the returned set as table OR as chart. This is still experimental but I suspect it would cover a similar role of grouping by any search result element (say, from_user or RURI) and stat a specific result as chart.

**Comment by negbie** [comment]
Combined with Grafanas 6.3 data links this could be very usefull.

**Comment by adubovikov** [comment]
https://www.ag-grid.com/javascript-grid-grouping/ Unfortunately this is an Enterprise feature. Let us check if there are external plugins for this

**Comment by adubovikov** [comment]
https://github.com/Servoy/aggridcomponents/wiki/Data-Grid

**Comment by adubovikov** [comment]
we can add this plugin later

**Comment by adubovikov** [comment]
H11 has been released
